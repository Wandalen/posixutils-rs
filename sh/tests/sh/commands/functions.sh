print_hello() {
    hello="Hello, World!"
    echo $hello
}
echo $?

print_args() {
    echo $@
    echo $*
    echo $#
    echo $1
    echo $2
    echo $3
}
echo $?

non_zero_return () {
  false
}

print_hello
echo $?
print_args a b c
echo $?
non_zero_return
echo $?
