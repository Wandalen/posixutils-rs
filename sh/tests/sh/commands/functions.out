0
0
Hello, World!
0
a b c
a b c
3
a
b
c
0
1
